### ISSUE TITLE
Question regarding the indices of validation set and training dataset

### ISSUE THREAD

**Issue by Oliverbansk** [ylabbe/robopose#3]
Hi! This is really fantastic work and I enjoyed reading your paper. Right now I have some questions regarding the indices of validation set and training set. 
In `robopose/training/train_articulated.py`, your sampler for the training dataset covers the whole indices of the training dataset while the sampler for the validation dataset ramdomly picks out 10% of the indices of a whole training dataset. I wonder if it will bring forth a data leak, that the validation set shares same data with the training set?
<img width="716" alt="Screenshot 2023-06-08 at 18 22 32" src="https://github.com/ylabbe/robopose/assets/104628021/3a01bd0d-3044-49bb-a71e-5f6f1baa237f">
I have printed out all the `view_id` fields in the frame_index when iterating with `ds_val_iter` and `ds_train_iter`, and I found most of the ids in the validation set is in the training set as well, which may results from the code above.
I'm wondering is there a specific cause for not splitting them completely? Maybe it is better to leverage all of the training set and the validation set is just for observation? We are dealing with the same dataset and we do feel that the size of the training set is a bit small. 
And if our obervation is correct, we are also wondering how did you prevent the overfitting problem during training (when the two set overlap with each other).
Thank you!! :)
